README.md

```diff
@@ -88,6 +88,10 @@ Alternatively, you can import the library directly from a CDN:
 -   [onSessionRenewed](#onSessionRenewed)
 -   [logout](#logout)
 -   [getTopazToken](#getTopazToken)
+-   [showAppRating](#showAppRating)
+-   [bottomSheet](#bottomSheet)
+-   [fetchPhoneNumbes](#fetchPhoneNumbes)
+-   [updatePhoneNumbers](#updatePhoneNumbers)
```

Comment:
maybe we could remove this markdown TOC, now GH has this feature built in:
![image](https://user-images.githubusercontent.com/1849576/197965674-cc2055a1-43b8-4d93-b396-62c154ce06af.png)

Reply:
Removed.